### ISSUE TITLE
Datamodel stub for fac object

### ISSUE THREAD

**Issue by martinhannigan** [peeringdb/peeringdb#1156]

**Comment by mustangthz** [comment]
I don't think that a webpage should be required for a FAC.

**Comment by martinhannigan** [comment]
@peeringdb/pc ping tks

**Comment by peterhelmenstine** [comment]
Website has historical precedence. Not sure what that reasoning was. Would be good to know the initial context.

**Comment by grizz** [comment]
I don't recall what the initial reasoning was behind requiring the URL, perhaps to get information of the facility?

Is there actually a facility that doesn't have a website?

**Comment by martinhannigan** [comment]
Yes. :) Well. It didn't. A website transition was taking place and rather
than waste 2K on an update, it was decided by the party that it can wait
for the new site. From what I understand it was to make sure that the
facility "was real". That's not working out too well at the moment. Its
also prescriptive.

I want to do this exclusively and not pay for web at all for example:
https://www.nc-ix.org/

**Comment by mustangthz** [comment]
Tons of facilities do not have actual webpages.

Leasing agencies may have a “website” for the building.

As we make the transition to the edge and Tier2 and Tier3 markets, we’re going to have more and more FACs popping up without webpages.

-Chris

**Comment by martinhannigan** [comment]
PDB V3 +1

**Comment by grizz** [comment]
Oh right, I think it was to help the AC deduce if a facility was real.

I find it hard to believe that a company owning a data center doesn't have a website, even if not facility specific. :)

So I would be good with removing website as a required field once the new process for approving facilities is put into place, @leovegoda can we link this issue to that project please?

Making the website entry optional would not require a v3 bump.

**Comment by mustangthz** [comment]
You’re thinking like someone that’s operated a real datacenter before :)

Think Wells building in MKE or Network 222 in Madison but with owners that know nothing about telecom.

It’s really common for these legacy carrier hotels to not have a “real” website, but have a page on the leasing agent’s site.

-Chris

**Comment by martinhannigan** [comment]
Agenda for next PC meeting to determine. Open a long time.

**Comment by martinhannigan** [comment]
My last thought on if we should add an ADDR object (or come up with some other fix)

Here's a small sample of FAC objects that understand you also improve findability with the address inserted into FAC name. That seems like a strong indicator that we should put further work into the issue.

Digital Realty SFO   (200 Paul)
Digital Realty NYC (60 Hudson)
Digital Realty NYC (111 8th Ave)
Digital Realty SFO (365 Main)
Equinix NY9 - New York, 111 8th Avenue
Digital Realty NYC (32 AofA)
Digital Realty ATL (56 Marietta)
Digital Realty PHX (120 E Van Buren)
Printer's Square Chicago (600 S Federal)
Netrality - Indy Telcom Center - 733 W.   Henry
FiberNet Telecom Group - 60 Hudson St
FiberNet Telecom Group New York (111   Eighth Ave)
Markley Group One Summer Street Boston
165 Halsey Meet-Me Room
Crown Castle Brentwood (1650 Islip)
Digital Realty DFW (2323 Bryan St)
Digital Realty LAX (600 W 7th)
Digital Realty IAD (44480 Hastings)
Netrality Houston - 1301 Fannin

HTH

**Comment by peterhelmenstine** [comment]
I put addresses in our facility names to enhance the search results however entries now show up in the beta search when you search by address so not sure it's needed when out of beta.

I was just going to start removing addresses from my facility names due to this search feature update.

**Comment by martinhannigan** [comment]
Glad we're getting it all sorted out, like search.

From an information organization perspective, I'm not sure search is a
substitute for hierarchy or how we would map FAC's to addresses without a
key like address. Search doesn't replace all the twisting people have done
(as an example, that list) to get noticed. It's also less equitable. Some
companies have people who do this full time and you can see that in the
entire list. Others, don't really have the resources to figure it out or
their not marketing keen - which is also what address is helpful for.

Creating an effective content hierarchy can help search (and search
engines) better understand and index the site which, in turn, leads to more
accuracy. Been awhile since I had to write any serious code, but also
strikes me a less resource and code intensive. YMMV.

HTH

**Comment by martinhannigan** [comment]
I removed the Digital Realty address hacks and grabbed another sampling.

1025Connect
1101 Stewart Ave, Garden City
The Pittock
1623 Farnam
740 Notre-Dame West Montreal
55 Marietta
801 Tenth Street Modesto (Ayera)
500 Green (AKA VOLICO 2.0)
Oak Tower
Cincinnatti Bell - West 7th Street 
Citynet Columbus
Citynet Pittsburgh
Citynet Indianapolis
Associated Engineering Plaza (10909   Jasper Avenue)
CoreSite - Los Angeles (LA1) One
Crown Castle Brentwood (1650 Islip)
801 Main Street - Lenoir
Epsilon Global Hubs New York (60 Hudson)
Equinix NY8 - New York, 60 Hudson
FiberNet Telecom Group - 60 Hudson St
FiberNet Telecom Group New York (111   Eighth Ave)
FiberNet Telecom Group Newark - 165   Halsey
180 E Broad St.
710 Tucker St Louis
INOC Albany (80 State St)
210 Sheppard Avenue East
3500 Garrott
151 Front Street West Toronto
321 S Boston Ave
360 Spear
Quonix Datacenter (2401 Locust St)
274 Brannan Street
The Franklin Exchange Tampa
800 south hope
165 Halsey Meet-Me Room
501 Franklin Ave
324 E Wisconsin
232 20th St NW, East Grand Forks
230 Congress
